bokehjs/src/coffee/models/plots/plot.coffee

```diff
@@ -14,6 +15,11 @@ PlotCanvas = require("./plot_canvas").Model
 class PlotView extends LayoutDOM.View
   className: "bk-plot-layout"
 
+  bind_bokeh_events: () ->
+    super()
+    title_msg = "Title object cannot be replaced. Try changing properties on title to update it after initialization."
+    @listenTo(@model, 'change:title', () => logger.warn(title_msg))
```

Comment:
Just to note that this is not new to title, its them same fundamental problem as not being able to swap out an axis on the fly.